### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by ppolakiewicz**
Some code cleanup detected by IntelliJ.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -26,124 +26,250 @@
       }
```
Comment (devdo-eu) [inline package-lock.json:26]: Are you 100% sure this file should be in the repository? To me it looks like some kind of automatically generated file, which is the same as expelling it from the repo :P

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
```
Comment (devdo-eu) [inline package.json:3]: Same comment as `package-lock.json` ;)

**Comment by ppolakiewicz** [comment]
Yeah, Im preaty sure that this two files should be in repo.

package.json - keeps list of dependencies that our project needs. 

package-lock.json - according to documentatio:
This file is intended to be committed into source repositories.
https://docs.npmjs.com/configuring-npm/package-lock-json.html

**Comment by devdo-eu** [comment]
Maybe... but a 6.5k lines of code changed makes me wonder...

**Comment by ppolakiewicz** [comment]
Welcome to JS world :D

**Pull request merged**